Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.6.5'
+ruby '2.7.0'
```

Comment:
I think we're probably ready to upgrade to use of 2.7.0 as a stable Ruby release across each of our Jekyll instances.

In order to do so completely, we'll want to update the Netlify config simultaneously. // @kushthedude

Reply:
Let's move to manual `netlify.toml` rather than using their UI Panel.